scripts/build-types.ts

```diff
@@ -0,0 +1,24 @@
+import cue from 'cuelang-js';
+import { default as openapiTS } from 'openapi-typescript';
+
+import { promises as fs } from 'fs';
+
+const cueToTSTypes = async (input: string) => {
+	const cueresult = await cue('export', [input], { '--out': 'openapi' });
+	if (cueresult.code !== 0) {
+		console.error(cueresult.stderr);
+		throw new Error('Error exporting cue as openapi');
+	}
+	const openapi = JSON.parse(cueresult.stdout);
+	return openapiTS(openapi);
+};
+
+(async () => {
```

Comment:
Declared function as `async` but never used `await`. I suggest rewriting the function body to use try/await/catch

Reply:
removed anonymous async function call in latest commit